### PR TITLE
Update README.md: dependencies

### PR DESCRIPTION
**Pull request by jerzybrzoska**
Added a note on dependencies.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,6 +147,13 @@ Summary
 
 ## Installation
 
+### Dependencies
+
+When installing with Cargo on Ubuntu you will need to have `libxrandr-dev` installed. To install it with apt run:
+```
+apt install libxrandr-dev
+```
```
Comment (pickfire) [inline README.md]: Can you please put this below as part of `From source`? And remove the heading.

**Comment by 9ary** [comment]
I'd also rather make this as distro-agnostic as possible. Building from source is for developers and packagers. As an end user you should ultimately bug your distro maintainers to package shotgun (or package it yourself an contribute it).

File: README.md (hunk 1 above)
Comment (jerzybrzoska) [reply README.md]: @pickfire I just did it.

File: README.md [hunk 2]
```diff
@@ -147,9 +147,10 @@ Summary
 
 ## Installation
 
+
 - From source:
   - install a recent Rust toolchain, libx11 and libxrandr (depending on your
-    distribution, you may need to install development headers separately)
+    distribution, you may need to install development headers separately). On Ubuntu you will need to have               `libxrandr-dev` installed. To install it with apt run `apt install libxrandr-dev`
```
Comment (pickfire) [inline README.md:153]: Can you please wrap this?

File: README.md [hunk 3]
```diff
@@ -147,9 +147,10 @@ Summary
 
 ## Installation
 
+
```
Comment (pickfire) [inline README.md:150]:
```suggestion
```

File: README.md (hunk 2 above)
Comment (9ary) [inline README.md:153]: Actually this is completely redundant (see my previous comment). I really don't want to clutter the readme with distro-specific information, only links to actual packages. Please ask the Ubuntu folks (or better yet, Debian) to package shotgun instead.

Comment (9ary) [reply README.md:153]: Once the package exists, I'll gladly accept a PR documenting that.

**Pull request closed without merging**